rs/tests/ckbtc/BUILD.bazel

```diff
@@ -0,0 +1,41 @@
+load("@rules_rust//rust:defs.bzl", "rust_library")
+
+package(default_visibility = ["//rs/tests:__subpackages__"])
+
+DEPENDENCIES = [
+    # Keep sorted.
+    "//rs/bitcoin/ckbtc/kyt",
+    "//rs/bitcoin/ckbtc/minter",
+    "//rs/canister_client",
+    "//rs/config",
+    "//rs/ledger_suite/icp:icp_ledger",
+    "//rs/ledger_suite/icrc1/ledger",
+    "//rs/nervous_system/common/test_keys",
+    "//rs/nns/common",
+    "//rs/nns/constants",
+    "//rs/nns/governance/api",
+    "//rs/nns/test_utils",
+    "//rs/registry/canister",
+    "//rs/registry/subnet_features",
+    "//rs/registry/subnet_type",
+    "//rs/rust_canisters/canister_test",
+    "//rs/rust_canisters/dfn_candid",
+    "//rs/tests/consensus/tecdsa/utils",
+    "//rs/tests/driver:ic-system-test-driver",
+    "//rs/types/base_types",
+    "//rs/types/management_canister_types",
+    "//rs/types/types",
+    "//rs/types/types_test_utils",
+    "@crate_index//:candid",
+    "@crate_index//:ic-agent",
+    "@crate_index//:ic-btc-interface",
+    "@crate_index//:slog",
+]
+
+rust_library(
+    name = "ckbtc",
+    srcs = glob(["src/**/*.rs"]),
+    crate_name = "ic_tests_ckbtc",
+    deps = DEPENDENCIES,
```

Comment:
I would just inline these DEPENDENCIES here if this is the only place this variable is referenced. Less jumping around.

Reply:
Almost every BUILD file in the ic repo that I've looked at has a DEPENDENCIES variable. I think we should be consistent with that (in general, and here in particular).

Also, I think defining DEPENDENCIES is a good idea for reasons beyond consistency: they help make sure that our BUILD files behave like our Cargo.toml files as much as possible.

Yes, this is "extra" indirection, but since we do this almost everywhere already, it shouldn't throw anyone off. If anything, not doing it here is more likely to mentally disrupt people. If we do not do DEPENDENCIES here, people will think, "Wait, why? Why is this a special snowflake? Is there some special reason that I do not see/understand?".

Reply:
I do think those other cases should also inline those variables in case they're referenced only once. Having a variable like this encourages using it multiple times which for dependencies is not a good idea because you risk depending on things you don't need and therefore prolonging build times. That''s not just an academic concern considering how many tests still depend on [DEPENDENCIES](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/tests/common.bzl?L7). 

OTOH I don't feel too strongly about this and it's also just my personal opinion. So feel free to ignore it.

The more important thing is the good work of inlining system-tests you started here which I really appreciate!